triedb/pathdb/disklayer.go

```diff
@@ -125,14 +125,14 @@ func (dl *diskLayer) Node(owner common.Hash, path []byte, hash common.Hash) ([]b
 			h := newHasher()
 			defer h.release()
 
-			got := h.hash(blob)
-			if got == hash {
-				cleanHitMeter.Mark(1)
-				cleanReadMeter.Mark(int64(len(blob)))
-				return blob, nil
-			}
-			cleanFalseMeter.Mark(1)
-			log.Error("Unexpected trie node in clean cache", "owner", owner, "path", path, "expect", hash, "got", got)
+			// got := h.hash(blob)
+			// if got == hash {
+			// 	cleanHitMeter.Mark(1)
+			// 	cleanReadMeter.Mark(int64(len(blob)))
+			return blob, nil
+			// }
+			// cleanFalseMeter.Mark(1)
+			// log.Error("Unexpected trie node in clean cache", "owner", owner, "path", path, "expect", hash, "got", got)
```

Comment:
@rjl493456442 hashing the node will not work for verkle, so I would like to skip it in that case. But I either have to pass the information that this is verkle somewhere, which would be messy, or somehow skip this test. Any advice for me here?

Reply:
fixed